app/services/notification/AmazonSimpleEmailServiceAsyncFactory.scala

```diff
@@ -0,0 +1,18 @@
+package services.notification
+
+import javax.inject.Inject
+
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.simpleemail._
+import com.google.inject.Provider
+
+
+class AmazonSimpleEmailServiceAsyncFactory @Inject()(awsCredentials :AwsCredentials) extends Provider[AmazonSimpleEmailServiceAsync]{
```

Comment:
Yes exactly...the gods required this! :godmode:  Maybe since there is the intention of moving to compile time DI maybe it is ok to leave it like that?!